### ISSUE TITLE
Nested StringLiterals isn't detected properly.

### ISSUE THREAD

**Issue by nanaya** [rubocop/rubocop#1014]
This code will pass default test:

``` ruby
puts "Here #{"be some string" if a == 1}"
```

...despite existence of double-quoted non-interpolated string.
